### DISCUSSION TITLE
Is there a way to detect if frigate stops getting a camera feed?

### DISCUSSION THREAD

**Discussion by patdemko** [blakeblackshear/frigate#1777] [Ask A Question] [answered]
I've been playing with Frigate for a short time, but already ran across an issue where a camera feed stopped showing up in the frigate gui (just a greenbox).  I then went to the camera's web gui and saw the video stream there also was not working which led me to the camera being the cause.  A reboot of the camera cleared it up, but I'm concerned it will come back.  Are there any type of MQTT messages or anything generated for when a camera feed stops in frigate?  I'm hoping I could use something like that to have home assistant notify me of the broken feed or to trigger a reboot of the camera via it's web api?

**Comment by lawm** [comment, marked as answer]
In Home Assistant you should have a `sensor.<camera_name>_fps` sensor.  Go to the Home Assistant -> History to see if it dropped to 0 when the camera had a green screen.  If so, you can create to create an automation based on this. 

Configuration -> Automations -> Create automation ->
Trigger type: State
Entity: sensor.camera_name_fps
To: 0   (0 fps)

**Reply by patdemko** [reply]
Not sure how I missed this, but it's exactly what I was looking for.  And it did drop to zero during the time of the issue so this should work great.  Thanks!
